### ISSUE TITLE
Extra sumo-gui pop-ups in visualizer

### ISSUE THREAD

**Issue by cathywu** [flow-project/flow#678]
## Feature Description

Currently, visualizing a policy (via `flow/visualize/visualizer_rllib.py`) with flow+rllib creates several (3?) extraneous instances of `sumo-gui` before the one that will actually play the policy. This is counter-intuitive to new users, and could cause them to believe that the visualizer is not working. Can we either avoid these instances or make clear in the documentation and/or the visualization script output that this is expected behavior?

cc @persimmon2014 in case there are more details I missed.

**Comment by kjang96** [comment]
+111

**Comment by eugenevinitsky** [comment]
Yeah, I hate this and have hated this for over two years now. Going to prioritize resolving this.

**Comment by Antonio-Dominguez** [comment]
Hi, have you already solved this or do you know how to do it?
I need that every time that I run a simulation Flow creates a single Sumo instance, but it creates 2 per simulation instead.

This is a problem since I am using a second TraCi client externally and I want to fix the port in which Sumo is listening, but I get an error every time the second instance is created since the first one is already listening in the fixed port. 

Any idea?

Thank you
